src/ServiceControlInstaller.Engine/Instances/TransportNames.cs

```diff
@@ -37,6 +37,7 @@ public static class TransportNames
         public const string RabbitMQClassicDirectRoutingTopology = "RabbitMQ - Direct routing topology (classic queues)";
 
         public const string RabbitMQQuorumDirectRoutingTopology = "RabbitMQ - Direct routing topology (quorum queues)";
+        public const string LearningTransport = "Learning Transport";
```

Comment:
None of the others include transport in the name

```suggestion
        public const string LearningTransport = "Learning";
```